### DISCUSSION TITLE
Advanced options box

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#20776] [Ideas]
There are a lot of small options or keybinds that don't/wouldn't really make sense in the full options or keybinds menu, but are still quite useful.
An advanced options box would allow for cutting down of bloated menus (removing options that are barely ever used, yet still useful for certain users) by allowing for text input instead of individual options.

### Things that could either be added or moved to the advanced options box:
- Touchscreen cursor hide
- joystick and MIDI input settings (which could be expanded upon to allow for more advanced settings as well) 
- separate hold to confirm activation times for exit and retry
- setting intro sequence to none
- combo break interval (ie how big of a combo you have to break to get the break sound, could be set differently for different gamemodes)
- an easy way to show UR and keys in HUD hide mode
- adding more keys to the disable list on top of the windows key (ie caps lock and num lock)
- setting different offsets for different mod combos in different gamemodes (HR and HT seem to screw with the offset a ton)
- UI scaling values below 0.8
- setting certain skins for certain gamemodes
- key combinations for all menu items in the editor, as well as other options that dont make sense to include in menus, but could be useful for certain users, such as send all selected hit objects to a new difficulty.
- keybinds for swapping gamemodes
- and all other options could also be set here as well
- tablet width and height outside of the limit

(not every option listed here has to be added/moved, and some may be duplicates of other suggestions, I'm just giving examples for how this could be used)
searching any of the above options (and any others added) would pull up the advanced options box (which would be accompanied by a help button, which would take the user to a help page with all the documentation on options)

Implementing this would allow for lesser used or needed features to be implemented without bloating menus.

**Comment by peppy** [comment]
Any of these which relate to gameplay elements will be in the skin editor, on the elements that they relate to.

Any you have proposed range changes would not be advanced. They would just be fixed to allow the required ranges.

**Reply by Doksha** [reply]
3 were related to skinning (4 if you count the intro), and 2 were related to ranges, there were a total of 8/7 other examples (remember, they arent solid and are more to just outline a point, not to serve as actual options that MUST be used.) 
Nothing you are saying has anything to do with an advanced options box, just to do with a few of the options I used as examples.
